### ISSUE TITLE
Missing metrics type "logstash_pipeline_plugins_outputs_bulk_*" and "logstash_pipeline_plugins_outputs_documents_*"

### ISSUE THREAD

**Issue by LHozzan** [alxrem/prometheus-logstash-exporter#21]
Hello.

I actually playing with this great project, but spotted a problem with these metrics. Sometimes are missing :(.

In Grafana it looks like:
![LogStashExporter-MissingMetric3](https://user-images.githubusercontent.com/24796361/124736591-767bd180-df17-11eb-9394-61a00249f46a.png)

When I attempt to watch LogStash Monitoring API, these metrics are still available:
![LogStashExporter-MissingMetric2](https://user-images.githubusercontent.com/24796361/124736728-93b0a000-df17-11eb-8ec7-8d9e345f4fd8.png)

When I attempt to watch LogStash Exporter metrics endpoint for Prometheus, sometimes are these metrics here, sometimes missing:
![LogStashExporter-MissingMetric](https://user-images.githubusercontent.com/24796361/124736850-b3e05f00-df17-11eb-805c-a44b7805b822.png)
![LogStashExporter-MissingMetric1](https://user-images.githubusercontent.com/24796361/124736852-b478f580-df17-11eb-88d4-0d4cbe4f0407.png)

But similar metrics `logstash_pipeline_plugins_outputs_events_*` working correctly.

Actually I spot only these two "family metrics" as problematic, but if I spot another, I will let you know.

---

_Used versions:_
Kubernetes: `v1.20.6`
LogStash Exporter: `0.7.0`
LogStash: `7.10.2`
Prometheus: `2.27.1`

**Comment by LHozzan** [comment]
Hello again.

Today I continue with playing and have some updates :).

**Problem was in scrape time.** We default use 30s for all collected metrics, but it seems, that LogStash Exporter have a problem with this value. If I lower it to 20s, metrics are totally random, sometimes we have it, sometimes it missing at all, sometimes come totally unusable metrics (default ms are showed in Grafana as a hours ...). After investigation I reach usable 15s for scrape.

I spotted another strange problem with raw metric too. If I set scrape time to 30s, some metrics have randomly zero value. In Grafana looks like "saw":
![LogStashExporter-saw](https://user-images.githubusercontent.com/24796361/124910456-e65b8c00-dfeb-11eb-8ef0-f9c3e59a2f1d.png)

If I change scrape time to 15s, same metric looks better:
![LogStashExporter-line](https://user-images.githubusercontent.com/24796361/124910531-fb381f80-dfeb-11eb-9217-41ff346b0948.png)

Exist chance, that problem isnt in LogStash Exporter as is. Maybe is in included libraries, I really dont know.

**Summary:** Workaround for this problem is change scrape time (for my case `30s` -> `15s`).

Hope, that helps!
